project1/codesubmission/clockwise/Robot.cc

```diff
@@ -0,0 +1,139 @@
+#include "Robot.h"
+#include <cstdlib>
+#include <iostream>
+#include <string>
+#include <limits> // std::numeric_limits
+/**Proj1 clockwise: Group10: Aguilar, Andrew, Kamel, Kennedy **/
```

Comment:
Looks good! I'm assuming he wants the data at the top of the roomba-roam.cc file instead of the Robot.cc one though. When able, just copy and paste it over!